etc/firejail-default

```diff
@@ -30,7 +30,8 @@ profile firejail-default flags=(attach_disconnected,mediate_deleted) {
 /{,var/}run/user/**/dconf/user rw,
 /{,var/}run/user/**/pulse/ rw,
 /{,var/}run/user/**/pulse/** rw,
-/{,var/}run/user/**/*.slave-socket rw,
+/{,var/}run/user/**/*.slave-socket rwl,
+/{,var/}run/user/**/#@{PID} rw,
```

Comment:
If I'm not mistaken everything behind '#' is just a comment. How much do we need this line, can we do without?

Reply:
correcting myself. It seems that '#' can be part of a pathname! But this is undocumented as far as I can see, so I'd still suggest to escape the '#'

Reply:
It catches things like /run/user/1000/#79965 which are needed. I modified it as you requested.